pycbc/events/stat.py

```diff
@@ -1715,13 +1715,16 @@ def rank_stat_single(self, single_info,
             The array of single detector statistics
         """
         sngls = single_info[1]
-
         ln_noise_rate = sngls['snglstat']
         ln_noise_rate -= self.benchmark_lograte
-        network_sigmasq = sngls['sigmasq']
-        network_logvol = 1.5 * numpy.log(network_sigmasq)
-        benchmark_logvol = sngls['benchmark_logvol']
-        network_logvol -= benchmark_logvol
+        if not np.isnan(sngls['benchmark_logvol']):
+            # Only include this term if the benchmark log volume is defined
+            # This should always be the case in the offline search, but may
+            # not be in the live search.
+            network_sigmasq = sngls['sigmasq']
+            network_logvol = 1.5 * numpy.log(network_sigmasq)
+            benchmark_logvol = sngls['benchmark_logvol']
+            network_logvol -= benchmark_logvol
```

Comment:
This doesn't seem quite right. If the` benchmark_logvol` is nan, it seems that the `network_logvol` variable is never set to any value. I think this would cause an error a few lines down when calculating the `loglr`. I think we just need to add an 
```
else:
    network_logvol = 0
```

Reply:
Good catch!